Command failed: /usr/bin/git checkout conf-override

When i install tap@13.0.0，it will report error.
```
npm ERR! Command failed: /usr/bin/git checkout conf-override
npm ERR! error: pathspec 'conf-override' did not match any file(s) known to git.
npm ERR!
```

--- a/package.json
+++ b/package.json
@@ -35,7 +35,7 @@
     "jackspeak": "^1.3.7",
     "minipass": "^2.3.5",
     "mkdirp": "^0.5.1",
-    "nyc": "github:isaacs/nyc#conf-override",
+    "nyc": "^14.1.0",
     "opener": "^1.5.1",
     "own-or": "^1.0.0",
     "own-or-env": "^1.0.1",

